### PR TITLE
batch processing mac/linux: adds find command - for #207

### PR DESCRIPTION
**Pull request by kieranjol**
Hi all,

I thought I'd take a stab at this. Looking from the comments in #207, I thought that it might be overkill to do a full, command by command breakdown of `find`, so I just threw in a one-liner. I already did something like this over here so I could just reuse some of this if needed: https://github.com/kieranjol/ifi_batch_workshop#recursive-loop-search-through-all-subfolders

Also, I couldn't figure out how to remove the .mov extension while using `find`. It could be done by piping the output of find to a for loop, but I thought this kept things a lot simpler.

Let me know what ye think anyhow,

### CODE REVIEW

**Review by ablwr — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1267,6 +1267,8 @@
           <p>The basic pattern will look similar to this:<br>
           <code>for item in *.ext; do ffmpeg -i $item <i>(ffmpeg options here)</i> "${item%.ext}_suffix.ext"</code></p>
           <p>e.g., if an input file is bestmovie002.avi, its output will be bestmovie002_suffix.avi.</p>
+          <p>Variation: recursively process all mxf files in subdirectories using <code>find</code> instead of <code>for</code>:</p>
```
Comment (retokromer) [inline index.html]: I would say `all MXF files` with capitals as it’s an abbreviation.

**Review by retokromer — APPROVED** [review]
Just one nit-picking, sorry…

File: index.html (hunk 1 above)
Comment (kieranjol) [reply index.html]: Agreed! I changed it just there. Thanks @retokromer & @ablwr for reviewing.

**Review by retokromer — APPROVED** [review]

**Comment by retokromer** [comment]
Thank you, @kieranjol!

**Comment by ablwr** [comment]
always the best!

**Pull request merged**